src/Microsoft.CodeQuality.Analyzers/UnitTests/Maintainability/AvoidUnusedPrivateFieldsTests.cs

```diff
@@ -307,7 +304,7 @@ public async Task CA1823_VisualBasic_FieldOffsetAttributeError_NoDiagnostic()
     <System.Runtime.InteropServices.FieldOffsetAttribute(8)> _
     Private fieldWithFieldOffsetAttribute As Integer
 End Class
-", CompilerDiagnostics.None);
+");
```

Comment:
📝 This test should be updated to match the C# test (it's not currently testing what it claims to be testing)

Reply:
@Evangelink the wrong test was updated here. I'm expecting a change to `CA1823_VisualBasic_FieldOffsetAttributeError_NoDiagnostic`.

Reply:
Fixed